packages/btp-utils/src/app-studio.ts

```diff
@@ -34,9 +34,9 @@ export function getAppStudioProxyURL(): string | undefined {
  */
 export async function getCredentialsForDestinationService(instance: string): Promise<string> {
     try {
-        const serviceInfo = await cfGetInstanceKeyParameters(instance);
-        const clientId = serviceInfo.uaa?.clientid || serviceInfo.clientid;
-        const clientSecret = serviceInfo.uaa?.clientsecret || serviceInfo.clientsecret;
+        const serviceCredentials = (await cfGetInstanceKeyParameters(instance)).credentials;
```

Comment:
Is it possible that `await cfGetInstanceKeyParameters(instance)` returns `undefined` or `null`?

Reply:
Yes, it is possible https://github.com/SAP/cloud-foundry-tools-api/blob/master/src/cf-local.ts#L592. I will update the PR.

Reply:
@tobiasqueck On second thought, if it returns `undefined`, then an error will be thrown that we can get credentials. Which is OK, right? I mean the whole flow can't work without credentials AFAIK.

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/547/commits/e1d0434d06b5a6cb816f3de2313a8892f58ab088